.github/workflows/node.js.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  compile-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: restore lerna
+      uses: actions/cache@v2
+      with:
+        path: |
+          node_modules
+          packages/**/node_modules
+        key: node_modules-${{ hashFiles('**/package-lock.json') }}
+
+    - name: Compile
+      run: |
+        npm install
+        npm run compile
+    - name: Lint & Typecheck
+      run: npm run check 
+    - name: Unit Test
+      run: npm run test:coverage
+    - name: Coverage
+      uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
question: where is this secret coming from? I don't see any configured in the repo settings

Reply:
GitHub actions uses a per job automatically generated token ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow https://github.com/coverallsapp/github-action